pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1771,10 +1802,14 @@ type DMClusterSpec struct {
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
-	// Base annotations of DM cluster Pods, components may add or override selectors upon this respectively
+	// Additional annotations for dm pod
```

Comment:
The annotations and labels defined here may also be overridden by the ones defined in the masterSpec or workerSpec.